geode-redis/src/main/java/org/apache/geode/redis/GeodeRedisServer.java

```diff
@@ -306,7 +306,7 @@
    * @return {@link RegionShortcut}
    */
   private static RegionShortcut setRegionType() {
-    String regionType = System.getProperty(DEFAULT_REGION_SYS_PROP_NAME, "PARTITION");
+    String regionType = System.getProperty(DEFAULT_REGION_SYS_PROP_NAME, "REPLICATE");
```

Comment:
Why the change to REPLICATE region type as the default? Should it be PARTITION_REDUNDANT, perhaps?

Reply:
i was just messing around...i'll rebase this change out of the PR